### PR TITLE
Added calculator for eighth homework

### PR DESCRIPTION
**Pull request by miqsargsyan21**

### CODE REVIEW

File: js/eighthHomework/script.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    numbers: [],
+    operators: []
+};
+
+let processWriting = (param) => {
+    if (data.newState) {
+        document.querySelectorAll("#result-div p")[0].innerText = "";
+        data.newState = false;
+    }
+    if (typeof param === typeof 5) {
```
Comment (akadevil) [inline js/eighthHomework/script.js:14]: @miqsargsyan21 jan kbacatres `typeof` inchi hamar es estex ogtagorcel?

File: js/eighthHomework/script.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    newState: true,
+    isPreviousOperator: false,
+    currentNumber: [],
+    numbers: [],
+    operators: []
+};
+
+let processWriting = (param) => {
+    if (data.newState) {
+        document.querySelectorAll("#result-div p")[0].innerText = "";
```
Comment (akadevil) [inline js/eighthHomework/script.js:11]: qani vor `document.querySelectorAll("#result-div p")` shat es ogtagorcum, karelia sa mitex popoxakani veragrel orinak senc `const resultBox = document.querySelectorAll("#result-div p");`  u heto `resultBox` n ogtagorces, btw nuyn code-n orinak henc `document.querySelectorAll("#result-div p")` senc dzevov amenur ogtagorceln kochvuma `code duplication`, ashxati xusapes
u karelia `querySelectorAll` i texn `querySelector` ogtagorcel, qani vor verjins 1 elementa veradarcnum hetevabar `[0]` grelu kariq chi lini

**Review by akadevil — COMMENTED** [review]
good job!

File: js/eighthHomework/script.js (hunk 1 above)
Comment (miqsargsyan21) [reply js/eighthHomework/script.js:14]: @akadevil vorpeszi stugem tiv a te che  nermucvacy :joy:

File: js/eighthHomework/script.js (hunk 2 above)
Comment (miqsargsyan21) [reply js/eighthHomework/script.js:11]: Shat lav.. uxxaki mtacumei te avelia lava avel popoxakan chpahem

File: js/eighthHomework/script.js (hunk 1 above)
Comment (akadevil) [reply js/eighthHomework/script.js:14]: isk `Number.isInteger()` porcel es?

Comment (akadevil) [reply js/eighthHomework/script.js:14]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isInteger

Comment (miqsargsyan21) [reply js/eighthHomework/script.js:14]: @akadevil chee :joy:

**Pull request merged**